pkg/controller/gslb/gslb_controller_test.go

```diff
@@ -631,6 +625,24 @@ func TestGslbController(t *testing.T) {
 			t.Errorf("got:\n %s Gslb ingress annotations,\n\n want:\n %s", ingress.Annotations, expectedAnnotations)
 		}
 	})
+
+	t.Run("Reflect GeoTag in the Status", func(t *testing.T) {
+		err = os.Setenv("CLUSTER_GEO_TAG", "eu")
+		if err != nil {
+			t.Fatalf("Can't setup env var: (%v)", err)
+		}
+		reconcileAndUpdateGslb(t, r, req, cl, gslb)
+		got := gslb.Status.GeoTag
+		want := "eu"
+
+		if got != want {
+			t.Errorf("got: '%s' GeoTag status, want:'%s'", got, want)
+		}
+		err = os.Unsetenv("CLUSTER_GEO_TAG")
```

Comment:
looks good. Just we should use  `defer <cleanup code...>` , otherwise there is potentional risk that `Unsetenv` never happens.